### PR TITLE
#CNV-33128: adjust tp tab section to web console overview

### PR DESCRIPTION
**Pull request by sabrinajess**
Version(s): 4.15+

Issue: [CNV-33128](https://issues.redhat.com//browse/CNV-33128)

Link to docs preview: https://69788--ocpdocs-pr.netlify.app/openshift-enterprise/latest/virt/getting_started/virt-web-console-overview#overview-page_virt-web-console-overview

QE review: No QE review requested. Updating docs location  that will not be specific to each TP feature, but a general note of where to find the TP features in the UI.

Additional information:

### CODE REVIEW

File: virt/getting_started/virt-web-console-overview.adoc [hunk 1]
```diff
@@ -79,13 +79,18 @@ The *Overview* page displays resources, metrics, migration progress, and cluster
...
+|The *Settings* tab contains the *Cluster* tab, *User* tab, and *Preview features* tab.
 
 |*Settings* -> xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-settings-cluster_virt-web-console-overview[*Cluster* tab]
 |{VirtProductName} version, update status, live migration, templates project, preview features, and load balancer service settings.
 
 |*Settings* -> xref:../../virt/getting_started/virt-web-console-overview.adoc#overview-settings-user_virt-web-console-overview[*User* tab]
 |Authorized SSH keys, user permissions, and welcome information settings.
+
+|*Settings* -> *Preview features*
+|Enable select link:https://access.redhat.com/support/offerings/techpreview/[preview features] in the UI. Features in this tab change frequently.
```
Comment (JoeAldinger) [inline virt/getting_started/virt-web-console-overview.adoc:91]: You refer to this as a tab in the sentence after it so I think this should be, `[*Preview features*] in the UI.`

**Review by JoeAldinger — COMMENTED** [review]
One small suggestion and don't forget to squash your commits.

File: virt/getting_started/virt-web-console-overview.adoc (hunk 1 above)
Comment (sabrinajess) [reply virt/getting_started/virt-web-console-overview.adoc:91]: Leaving this as-is. This "preview features" is referring to preview features in general and not to the tab itself.

**Pull request merged**